metadata/src/main/resources/common/metadata/AccessControlRecord.json

```diff
@@ -1,38 +0,0 @@
-// Licensed to the Apache Software Foundation (ASF) under one or more
-// contributor license agreements.  See the NOTICE file distributed with
-// this work for additional information regarding copyright ownership.
-// The ASF licenses this file to You under the Apache License, Version 2.0
-// (the "License"); you may not use this file except in compliance with
-// the License.  You may obtain a copy of the License at
-//
-//    http://www.apache.org/licenses/LICENSE-2.0
-//
-// Unless required by applicable law or agreed to in writing, software
-// distributed under the License is distributed on an "AS IS" BASIS,
-// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-// See the License for the specific language governing permissions and
-// limitations under the License.
-
-{
-  "apiKey": 6,
```

Comment:
Could we not delete this file in this PR? I think it'd be better we raise another JIRA or PR to delete it. I'm not 100% sure we will use it or not. Let's focus on missing record type in this PR. WDYT?

Reply:
Yeah, this makes sense, it's reasonable to keep it.